src/python/responseTimeTests.py

```diff
@@ -81,7 +122,7 @@ def captureEnv(logsDir):
   if svnRev.endswith('M'):
     if system('svn diff %s > %s/lucene.diffs 2>&1' % (LUCENE_HOME, logsDir)):
       raise RuntimeError('svn diff failed')
-    os.chmod('%s/lucene.diffs' % logsDir, o0444)
+    os.chmod('%s/lucene.diffs' % logsDir, 0o444)
```

Comment:
Good grief.  Did python2 accept o-prefix for octal number?

Reply:
This one may have been my fault in a previous PR? the way the octals were done was breaking things at a syntactic level... so I'm afraid I temporarily introduced the bug. But now we've made enough progress that checker finds such bugs.